api/server.js

```diff
@@ -37,3 +40,49 @@ const port = APP_CONFIG.PORT || 3000;
 app.listen(port, () =>
   console.log(`Server started on http://localhost:${port}`)
 );
+
+let runMonitor = false;
+
+const monitor = () => {
+  if (runMonitor) {
+    console.log(`[${dayjs().utc()}] Running monitor`);
+
+    const allInstances = db.instances.find({});
+    const truantInstances = allInstances.filter((instance) => {
+      if (typeof instance.instanceLastHealthyAt === "undefined") return false;
```

Comment:
Are you wanting to do a type check or do you just want to see if the value is undefined?

Would this do what you need?
```suggestion
      if (instance.instanceLastHealthyAt === undefined) return false;
```